js/main.js

```diff
@@ -19,6 +19,94 @@ function submission(event) {
   formObject.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(formObject);
-  $image.src = 'images/placeholder-image-square.jpg';
+  $newImage.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+  getEntry(data.entries[0]);
+  toggleNoEntries();
+  viewEntries();
+}
+
+function renderEntry(entry) {
+  var $entriesImg = document.getElementById('img' + entry.entryId);
+  var $entriesH2 = document.getElementById('h2' + entry.entryId);
+  var $entriesP = document.getElementById('p' + entry.entryId);
+  $entriesImg.src = entry.imageUrl;
+  $entriesH2.textContent = entry.title;
+  $entriesP.textContent = entry.notes;
+}
+
+document.addEventListener('DOMContentLoaded', getEntry('contentLoad'));
```

Comment:
This is calling `getEntry` **_immediately_**, you are not passing it as an event handler. This is really bad news.